### PR TITLE
Source new aws opts in integration tests

### PR DESCRIPTION
**Pull request by ravjotbrar**
### Summary

Updating Main.Scala in functional-tests to load in new aws config options properly. 

### Description

Wrapped all aws options in the new AWSArg type. Also, included requirement of Java8 or Java11 in the README. This branch also contains all of Jonathan's changes, which adds support for connecting to AWS S3 using 2FA and using IAM roles and also adds documentation around the new options added to enable this support.

### Related Issue

http://jira.verticacorp.com:8080/jira/browse/VER-77396  
http://jira.verticacorp.com:8080/jira/browse/VER-76344

### Additional Reviewers

@alexr-bq 
@jonathanl-bq

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,18 +50,13 @@ However you set up HDFS, Vertica needs to have a copy of the hadoop configuratio
 ```shell
 ALTER DATABASE <database name> SET HadoopConfDir = '/hadoop/conf/location/';
 ```
+### Java 
 
-### AWS S3
-
-To use the connector with S3, you will need to add the `hadoop-aws` dependency to your project:
-```libraryDependencies += "org.apache.hadoop" % "hadoop-aws" % "3.2.0"```
+The current versions of Java that are compatible with Spark are Java8 and Java11.
```
Comment (alexey-temnikov) [inline README.md:55]: ->  Spark Connector requires Java 8 (8u92 or later) or Java 11.

Comment (ravjotbrar) [reply README.md]: Changed.

File: functional-tests/src/main/resources/application.conf [hunk 2]
```diff
@@ -9,5 +9,6 @@ functional-tests {
   tlsmode="disable"
   truststorepath="/truststore.jks"
   truststorepassword="dbadmin"
+
```
Comment (alexey-temnikov) [inline functional-tests/src/main/resources/application.conf:12]: .nit extra space

Comment (ravjotbrar) [reply functional-tests/src/main/resources/application.conf]: Done

**Review by alexr-bq — APPROVED** [review]

**Review by alexey-temnikov — APPROVED** [review]

**Pull request merged**